db/models.py

```diff
@@ -50,3 +55,80 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        to=settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return (str(self.created_at).split("."))[0]
```

Comment:
```suggestion
        return str(self.created_at)
```

Reply:
This updated code will achieve the same result as your original code, but it's cleaner and more straightforward by directly converting self.created_at to a string without splitting and formatting